ignite/services/chain/runtime.go

```diff
@@ -47,17 +35,31 @@ func (p *stargatePlugin) Gentx(ctx context.Context, runner chaincmdrunner.Runner
 	)
 }
 
-func (p *stargatePlugin) Configure(homePath string, cfg *chainconfig.Config) error {
-	if err := p.appTOML(homePath, cfg); err != nil {
+// Start wraps the "appd start" command to begin running a chain from the daemon
+func (c Chain) Start(ctx context.Context, runner chaincmdrunner.Runner, cfg *chainconfig.Config) error {
+	validator := cfg.Validators[0]
+	servers, err := validator.GetServers()
+	if err != nil {
+		return err
+	}
+
+	err = runner.Start(ctx, "--pruning", "nothing", "--grpc.address", servers.GRPC.Address)
+
+	return &CannotStartAppError{runner.Cmd().Name(), err}
```

Comment:
I got confused with this part. It actually works because the command will block until the context is canceled which will be handled further down the line, it would be good to have a comment explaining this behaviour or a refactor to make it clear because one would expect a nil error if all works fine.